mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/resourcecluster/ExecutorStateManagerImpl.java

```diff
@@ -116,6 +116,7 @@ private boolean tryMarkAvailable(TaskExecutorID taskExecutorID, TaskExecutorStat
             log.debug("Marking executor {} as available for matching.", teHolder);
             double cpuCores = state.getRegistration().getMachineDefinition().getCpuCores();
             if (!this.executorByCores.containsKey(cpuCores)) {
+                log.info("[executorByCores] adding {} from TE: {}", cpuCores, teHolder);
```

Comment:
log.debug instead?

Reply:
this is rare e.g only happens with new cores.